src/webportal/src/app/job/job-view/fabric/task-attempt.jsx

```diff
@@ -0,0 +1,114 @@
+// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+import React, { useEffect, useState } from 'react';
+import { Stack, ActionButton, Text } from 'office-ui-fabric-react';
+import ReactDOM from 'react-dom';
+import { isNil, capitalize } from 'lodash';
+import { DateTime } from 'luxon';
+
+import { SpinnerLoading } from '../../../components/loading';
+import TaskAttemptList from './task-attempt/task-attempt-list';
+import { fetchTaskStatus } from './task-attempt/conn';
+import StatusBadge from '../../../components/status-badge';
+
+const params = new URLSearchParams(window.location.search);
+const userName = params.get('username');
+const jobName = params.get('jobName');
+const jobAttemptIndex = params.get('jobAttemptIndex');
+const taskRoleName = params.get('taskRoleName');
+const taskIndex = params.get('taskIndex');
+
+const TaskAttemptPage = () => {
+  const [loading, setLoading] = useState(true);
+  const [taskStatus, setTaskStatus] = useState(null);
+
+  useEffect(() => {
+    fetchTaskStatus(
+      userName,
+      jobName,
+      jobAttemptIndex,
+      taskRoleName,
+      taskIndex,
+    ).then(data => {
+      setTaskStatus(data);
+      setLoading(false);
+    });
+  }, []);
+
+  return (
+    <div>
+      {loading && <SpinnerLoading />}
+      {!loading && (
+        <Stack styles={{ root: { margin: '30px' } }} gap='l1'>
+          <ActionButton
+            iconProps={{ iconName: 'revToggleKey' }}
+            href={`job-detail.html?username=${userName}&jobName=${jobName}`}
+          >
+            Back to Job Detail
+          </ActionButton>
+          <Stack horizontal gap='l1' padding='s2'>
+            <Stack gap='m'>
+              <Text>Task Role</Text>
+              <Text>{taskRoleName}</Text>
+            </Stack>
+            <Stack gap='m'>
+              <Text>Task Index</Text>
+              <Text>{taskIndex}</Text>
+            </Stack>
+            <Stack gap='m'>
+              <Text>Task Uid</Text>
+              <Text>{taskStatus.taskUid}</Text>
+            </Stack>
+            <Stack gap='m'>
+              <Text>Task State</Text>
+              <StatusBadge status={capitalize(taskStatus.taskState)} />
+            </Stack>
+            <Stack gap='m'>
+              <Text>Task Retries</Text>
+              <Text>{taskStatus.retries}</Text>
+            </Stack>
+            <Stack gap='m'>
+              <Text>Task Start Time</Text>
+              <Text>
+                {DateTime.fromMillis(taskStatus.createdTime).toLocaleString(
+                  DateTime.DATETIME_MED,
+                )}
+              </Text>
+            </Stack>
+            <Stack gap='m'>
+              <Text>Task Complete Time</Text>
+              <Text>
```

Comment:
Align with job page? i.e. change to duration ?

Reply:
done